### PR TITLE
feat: Inbox management console

### PR DESCRIPTION
**Pull request by pranavrajs**
This is the continuation of the design update PR. This changes the design for the inbox pages. 

| Light | Dark |
| -- | -- |
| Screenshot 2024-08-29 at 9 59 20 AM | Screenshot 2024-08-29 at 9 59 31 AM|

### CODE REVIEW

**Review by muhsin-k — APPROVED** [review]
LGTM

File: app/javascript/dashboard/routes/dashboard/settings/inbox/components/ChannelName.vue [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+const twilioChannelName = () => {
+  if (props.medium === 'whatsapp') {
+    return t(`INBOX_MGMT.CHANNELS.WHATSAPP`);
+  }
+  return t(`INBOX_MGMT.CHANNELS.TWILIO_SMS`);
+};
+
+const readableChannelName = computed(() => {
+  if (props.channelType === 'Channel::Api') {
+    return globalConfig.apiChannelName || t('API');
```
Comment (iamsivin) [inline app/javascript/dashboard/routes/dashboard/settings/inbox/components/ChannelName.vue:42]: Translation is wrong, I guess. 
```suggestion
    return globalConfig.apiChannelName || t('INBOX_MGMT.CHANNELS.API');
```
image

**Review by iamsivin — COMMENTED** [review]
@pranavrajs Added some comments, could you please take a look?

File: app/javascript/dashboard/routes/dashboard/settings/inbox/components/ChannelName.vue (hunk 1 above)
Comment (pranavrajs) [reply app/javascript/dashboard/routes/dashboard/settings/inbox/components/ChannelName.vue:42]: Good catch. The fallback text was shown in this case.

**Comment by pranavrajs** [comment]
@iamsivin I've fixed the comments, please take another look.

**Review by iamsivin — APPROVED** [review]
LGTM

File: app/javascript/dashboard/routes/dashboard/settings/inbox/components/ChannelName.vue [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+const twilioChannelName = () => {
+  if (props.medium === 'whatsapp') {
+    return t(`INBOX_MGMT.CHANNELS.WHATSAPP`);
+  }
+  return t(`INBOX_MGMT.CHANNELS.TWILIO_SMS`);
+};
+
+const readableChannelName = computed(() => {
+  if (props.channelType === 'Channel::Api') {
+    return globalConfig.value.apiChannelName || t('API');
```
Comment (iamsivin) [inline app/javascript/dashboard/routes/dashboard/settings/inbox/components/ChannelName.vue:42]: @pranavrajs Reverted this change in this [commit](https://github.com/chatwoot/chatwoot/pull/10043/commits/98105173b6721e42761452fe1b1f59f438c87182)
```suggestion
    return globalConfig.value.apiChannelName || t('INBOX_MGMT.CHANNELS.API');
```

**Pull request merged**